<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SMTP-клиент для отправки почтовых сообщений</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Рабочая панель</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>SMTP-сервер:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="serverField">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Отправитель</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="senderField">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Файл пароля</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passFile">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Получатель</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="receiverField">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Копии</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="copiesList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Тема</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="subjectField">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="messageField">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>241</width>
       <height>201</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="sendButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Отправить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clearButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>480</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Очистить</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>261</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>Административная панель</string>
    </property>
    <widget class="QListWidget" name="groupsList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>261</width>
       <height>191</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Группы рассылок</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="groupField">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>240</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="groupButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Группа</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="emailField">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="emailButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Адрес</string>
     </property>
    </widget>
    <widget class="QListWidget" name="journalList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>300</y>
       <width>261</width>
       <height>211</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
